xml/cap_troubleshooting.xml

```diff
@@ -559,6 +559,36 @@ recover switch to commit or rollback pending transactions.
      </step>
     </substeps>
    </step>
+   <step>
+    <para>
+     If the <literal>mysql</literal> pod is in state READY: 0/1 and STATUS: Running, like this:
+    </para>
+<screen>
+&prompt.user;kubectl get pods -n uaa
+NAME                          READY STATUS    RESTARTS AGE
+configgin-helper-0            1/1   Running   0        16m
+mysql-0                       0/1   Running   0        16m
+mysql-proxy-0                 1/1   Running   0        16m
+post-deployment-setup-1-lg7vl 0/1   Completed 0        16m
+secret-generation-1-fczw4     0/1   Completed 0        16m
+uaa-0                         0/1   Running   0        16m 
+</screen>
+    <para>
+     Verify if all files of the <literal>mysql-0</literal> pod in
```

Comment:
Might need to reword a bit. Right now it feels like this can be interpreted as "there are files in the /var/vcap/store directory that belong to the mysql pod (but could also belong to other pods), check the ones that belong to the mysql pod are owned by the vcap user".  

Maybe something like:
- Verify all files in the `/var/vcap/store` of the `mysql-0` pod are owned by the `vcap` user
- Verify all files in the `mysql-0` pod's `/var/vcap/store` directory are owned by the `vcap` user